src/payment/strategies/bolt/bolt-payment-strategy.ts

```diff
@@ -11,12 +11,15 @@ import PaymentMethodActionCreator from '../../payment-method-action-creator';
 import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
 import PaymentStrategy from '../payment-strategy';
 
-import { BoltCheckout, BoltTransaction } from './bolt';
+import { BoltCheckout, BoltEmbedded, BoltTransaction } from './bolt';
 import BoltScriptLoader from './bolt-script-loader';
 
 export default class BoltPaymentStrategy implements PaymentStrategy {
     private _boltClient?: BoltCheckout;
+    private _boltEmbedded?: BoltEmbedded;
+    private _embeddedField?: any;
```

Comment:
Would like to understand the use of `any` here?

Reply:
Oh, my bad 😥 
Already fixed this thing